src/Common/src/Interop/Unix/System.Native/Interop.Terminal.cs

```diff
@@ -0,0 +1,43 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.InteropServices;
+using Microsoft.Win32.SafeHandles;
+using System.IO.Ports;
+
+internal static partial class Interop
+{
+    internal static partial class Sys
+    {
+        [DllImport(Libraries.SystemNative, EntryPoint = "SystemNative_Termios_Reset", SetLastError = true)]
```

Comment:
nit: it seems that the convention is to use just one underscore after SystemNative and the remaining Pascal casing.